installer-job/README.md

```diff
@@ -0,0 +1,33 @@
+# gk-installer
+Installer for [gluster/gluster-kubernetes](https://github.com/gluster/gluster-kubernetes)
+
+## Usage
+The installer expects that you are running the Job in the Kubernetes cluster you intend to deploy gluster-kubernetes to. It also expects that you will mount your [topology file](https://github.com/heketi/heketi/wiki/Setting-up-the-topology) to /etc/gk-deploy/topology.json (i.e. via ConfigMap)
+
+The repo includes an example ConfigMap and Job for deployment
```

Comment:
Mention that the ConfigMap we provide, as is, works with our example vagrant environment.